text-utility/text-utility.css

```diff
@@ -0,0 +1,62 @@
+.head {
+  color: #000;
+  word-break: break-all;
+}
+
+.h1-head {
+  font-size: 3rem;
+}
+
+.h2-head {
+  font-size: 2.5rem;
+}
+
+.h3-head {
+  font-size: 2rem;
+}
+
+.h4-head {
+  font-size: 1.75rem;
+}
+
+.h5-head {
+  font-size: 1.5rem;
+}
+
+.lg-text {
```

Comment:
You can also font-weight and line-height for the text .

Reply:
Done.. For heading i have added the font-weight separately..